eZ/Publish/Core/FieldType/Tests/ImageTest.php

```diff
@@ -817,6 +823,56 @@ public function provideInvalidDataForValidate()
                     ),
                 ],
             ],
+
+            // alternative text is null
+            [
+                [
+                    'validatorConfiguration' => [
+                        'AlternativeTextValidator' => [
+                            'required' => true,
+                        ],
+                    ],
+                ],
+                new ImageValue(
+                    [
+                        'id' => $this->getImageInputPath(),
+                        'fileName' => basename($this->getImageInputPath()),
+                        'fileSize' => filesize($this->getImageInputPath()),
+                        'alternativeText' => null,
+                        'uri' => '',
+                    ]
+                ),
+                [
+                    new ValidationError(
+                        'Alternative text is required.', null, [], 'alternativeText'
+                    ),
+                ],
+            ],
+
+            // alternative text is empty string
+            [
```

Comment:
I think it makes sense for these cases to have this comment as part of the key for the array. Even if the previous are not "keyed", it's still possible PHP-wise to add set keys explicitly here, so previous ones do not need to be updated if you don't want to.

```suggestion
            'alternative text is empty string' => [
```